### ISSUE TITLE
Add pipeline documentation for vacancy data

### ISSUE THREAD

**Issue by ODILMichelle** [open-innovations/yff-data#159]

**Comment by taz-chiles** [comment]
I have added pipeline documentation for vacancies here: https://data.youthfuturesfoundation.org/dashboard/vacancies/pipeline/ 

The exact methods will likely still change but we can make a separate ticket to do this once the pipelines have been finalised. 

Let me know any comments or questions about what I've written @ODILMichelle

**Comment by ODILMichelle** [comment]
I don't know enough about how y'all are processing the data - so if its in the same format as the other pages and both correct & complete, I'm happy for you to say it's done!

**Comment by luke-strange** [comment]
![Image](https://github.com/open-innovations/yff-data/assets/92686634/c45f34af-a9f9-48f5-8973-75efdf81722f)

Maybe reword "process them into a single csv extract" to "... and formats them into a single csv file"

**Comment by gilesdring** [comment]
Going to close this - we need to review and ticket per @taz-chiles suggestion above - I suspect that @luke-strange comment above will be less relevant given changes in extracting data?
